app/src/main/java/org/mozilla/fenix/search/SearchDialogFragment.kt

```diff
@@ -645,6 +665,40 @@ class SearchDialogFragment : AppCompatDialogFragment(), UserInteractionHandler {
         }
     }
 
+    /**
+     * Updates the search selector menu with the given list of available search engines.
+     *
+     * @param searchEngines List of [SearchEngine] to display.
+     */
+    private fun updateSearchSelectorMenu(searchEngines: List<SearchEngine>) {
+        searchSelectorMenu.menuController.submitList(
+            searchEngines.map {
```

Comment:
I think we should add `searchSelectorMenu.menuItems()` first then the `searchEngines`.  This way the settings menu will be on top as the design.